scripts/run_tasks.sh

```
@@ -0,0 +1,29 @@
+#   read_dir = os.environ['INTERMEDIATE']
+#   n = 0
+#   for split in [train_split, tuning_split, held_out_split]:
+#       for f in os.listdir(f'{read_dir}/data/{split}'):
+#           if f.endswith('.parquet'): n += 1
+#   print(n - 1)
+#   ")
+#
+#   # Then submit:
+#   sbatch --array=0-$N scripts/run_tasks.sh
+
+cd ~/EveryQuery
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)